src/components/dumb/OeGrid.vue

```diff
@@ -12,6 +12,10 @@ import { AgGridVue } from 'ag-grid-vue3';
 @import 'pyoes/scss/pyoes-settings';
 
 .ag-grid-vue {
+  .ag-ltr .ag-header-cell::after {
+    display: none;
```

Comment:
Probleem hiermee is wel dat als je een kolom `resizable` maakt, je dan geen indicatie meer hebt waar te klikken voor de resizen.

Komt er miss een class bij op als je een kolom resizable maakt zodat je dan dit streepke wel kunt tonen?

Reply:
Yes, die indicatie was inderdaad weg. Zou gefixed moeten zijn